appcell-resources/js/utils.js

```diff
@@ -207,6 +222,38 @@ ut.getDefaultImage = function (cellUrl, cellType) {
     return ut.getJdenticon(cellUrl);
 }
 
+/*
+ * A function that creates a button for a modal footer part
+ * @param id OK button id
+ * @param callback Processing when the OK button is pressed
+ */
+ut.createModalFooterTag = function (id, callback) {
+    let footerTag = $("<div>", {
+        class: "modal-footer"
+    });
+    let cancelBtnTag = $("<button>", {
+        type: "button",
+        class: "btn btn-default",
+        "data-dismiss": "modal",
+        "data-i18n": "Cancel"
+    });
+    footerTag.append(cancelBtnTag);
+
+    let okBtnTag = $("<button>", {
+        type: "button",
+        class: "btn btn-primary",
+        id: id
+    }).html("OK");
+    okBtnTag.click(function () {
```

Comment:
Click event can also be defined during the initialization.  
Example:  

    let aTag = $('<a>', {
        class: 'allToggle prev-icon',
        href: '#',
        onClick: 'moveBackahead();return false;'
    });